src/Views/TryInstallView.vala

```diff
@@ -27,63 +27,65 @@ public class TryInstallView : AbstractInstallerView {
-        nochanges_emblem.halign = Gtk.Align.END;
-
-        var nochanges_overlay = new Gtk.Overlay ();
-        nochanges_overlay.add_overlay (nochanges_image);
-        nochanges_overlay.add_overlay (nochanges_emblem);
-        nochanges_overlay.halign = Gtk.Align.CENTER;
-        nochanges_overlay.height_request = 51;
-        nochanges_overlay.width_request = 60;
-
-        var nosaving_label = new Gtk.Label (_("Any changes you make in Demo Mode will not be saved."));
-        nosaving_label.valign = Gtk.Align.START;
-        nosaving_label.max_width_chars = 40;
-        nosaving_label.wrap = true;
-        nosaving_label.set_justify (Gtk.Justification.CENTER);
-
-        var nosaving_image = new Gtk.Image.from_icon_name ("document-revert", Gtk.IconSize.DIALOG);
-
-        var return_label = new Gtk.Label (_("You can always return from Demo Mode to the installer by selecting the Install icon."));
-        return_label.max_width_chars = 40;
+        title_label.valign = Gtk.Align.START;
+        title_label.get_style_context ().add_class ("h2");
+
+        var choice_image = new Gtk.Image.from_icon_name ("computer-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
+        choice_image.halign = Gtk.Align.END;
+        choice_image.valign = Gtk.Align.START;
+
+        var choice_label = new Gtk.Label (_("You can install %s on this device now, or try Demo Mode without installing.").printf (Utils.get_pretty_name ()));
+        choice_label.halign = Gtk.Align.START;
+        choice_label.max_width_chars = 52;
+        choice_label.valign = Gtk.Align.START;
+        choice_label.wrap = true;
+        choice_label.xalign = 0;
+
+        var implications_image = new Gtk.Image.from_icon_name ("document-revert-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
+        implications_image.halign = Gtk.Align.END;
+        implications_image.valign = Gtk.Align.START;
+
+        var implications_label = new Gtk.Label (_("In Demo Mode, changes you make will not be saved and data from your previous operating system will be unchanged."));
+        implications_label.halign = Gtk.Align.START;
+        implications_label.max_width_chars = 52;
+        implications_label.valign = Gtk.Align.START;
+        implications_label.wrap = true;
+        implications_label.xalign = 0;
+
+        var return_image = new Gtk.Image.from_icon_name ("go-home-symbolic", Gtk.IconSize.LARGE_TOOLBAR);
+        return_image.halign = Gtk.Align.END;
+        return_image.valign = Gtk.Align.START;
+
+        var return_label = new Gtk.Label (_("You can always return to the installer from Demo Mode by selecting the Install icon."));
+        return_label.halign = Gtk.Align.START;
+        return_label.max_width_chars = 52;
         return_label.valign = Gtk.Align.START;
         return_label.wrap = true;
-        return_label.set_justify (Gtk.Justification.CENTER);
+        return_label.xalign = 0;
+
+        var grid = new Gtk.Grid ();
+        grid.orientation = Gtk.Orientation.VERTICAL;
```

Comment:
If you're just using `attach` and not `add` then setting orientation here doesn't really do anything